src/AddTodoForm.js

```diff
@@ -1,11 +1,19 @@
 import react from "react";
 function AddTodoForm() {
+    const handleAddTodo = (event) => {
+        event.preventDefault();
+        setNewTodo(event.target.value);
+        console.log(event.target.value);
+        handleAddTodo.remove()
```

Comment:
To reset the input, we will need to call `event.target.reset()`, because the input element is the target of the event the this function is handling.